packages/blob-storage-manager/src/env.ts

```diff
@@ -1,30 +1,61 @@
 import { z } from "zod";
 
-export const env = z
-  .object({
-    BEE_DEBUG_ENDPOINT: z.string().url().optional(),
-    BEE_ENDPOINT: z.string().url().optional(),
-    CHAIN_ID: z
-      .string()
-      .min(1)
-      .default("7011893055")
-      .transform((value, ctx) => {
-        const chainId = parseInt(value, 10);
-
-        if (isNaN(chainId) || chainId <= 0) {
-          ctx.addIssue({
-            code: z.ZodIssueCode.custom,
-            message: "CHAIN_ID must be a number greater than 0",
-          });
-
-          return z.NEVER;
-        }
-
-        return chainId;
-      }),
-    GOOGLE_STORAGE_BUCKET_NAME: z.string().default("blobscan-test-bucket"),
-    GOOGLE_STORAGE_PROJECT_ID: z.string().optional(),
-    GOOGLE_SERVICE_KEY: z.string().optional(),
-    NODE_ENV: z.enum(["development", "test", "production"]).optional(),
-  })
-  .parse(process.env);
+function booleanTransformer(varName: string) {
+  return (arg: string, ctx: z.RefinementCtx): boolean => {
+    const arg_ = arg.toLowerCase();
+    if (arg_ !== "true" && arg_ !== "false") {
+      ctx.addIssue({
+        code: z.ZodIssueCode.custom,
+        message: `${varName} must be either true or false`,
+      });
+
+      return z.NEVER;
+    }
+
+    return arg_ === "true";
+  };
+}
+
+const envSchema = z.object({
+  BEE_DEBUG_ENDPOINT: z.string().url().optional(),
+  BEE_ENDPOINT: z.string().url().optional(),
```

Comment:
can we check somewhere that either BEE_ENDPOINT or BEE_DEBUG_ENDPOINT is defined when `SWARM_STORAGE_ENABLED=false`?

Reply:
You mean when `SWARM_STORAGE_ENABLED=true`? That check is performed [here](https://github.com/Blobscan/blobscan/blob/48573d9f163375f59ace9b6d907710d3f779c50a/packages/blob-storage-manager/src/storages/SwarmStorage.ts#L70)

Reply:
Nope, let me rephrase.

BEE_ENDPOINT is required by the web app while BEE_DEBUG_ENDPOINT is required by the indexer.

We should warn the user in case `SWARM_STORAGE_ENABLED=true` but BEE_DEBUG_ENDPOINT and BEE_ENDPOINT are undefined.

Reply:
Got it. Let's require users to provide at least one of these endpoints to instantiate the storage and throw error in case any of the operations which requires the unset client is called.

Also, let's add a similar warning to the google storage as well